.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'DEVNOTES.md'
+      - 'CODE_OF_CONDUCT.md'
+      - 'LICENSE'
+      - '.gitignore'
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

Comment:
How about Python 3.9.x?

Reply:
This is a linter step. Why Python 3.9?

Reply:
0. Why to use older version when we can use newer one?
1. Pretty soon we will add `mypy` to the linter step and and it will be nice to use more concise typing, e.g.  `list[str]`  instead of `typing.List[str]`, by the first one is not supported in 3.7